docs/docs.md

```diff
@@ -0,0 +1,585 @@
+      })[]
+}[]
+```
+
+# `/event/{eventKey}/match/{matchKey}/stats`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  alliance: "red" | "blue"
+  team: string
+  teleop: (
+    | {
+        attempts: {
+          max: number
+          avg: number
+        }
+        successes: {
+          max: number
+          avg: number
+        }
+        statName: string
+      }
+    | {
+        // total
+        attempts: number
+        // total
+        successes: number
+        statName: string
+      })[]
+  auto: (
+    | {
+        attempts: {
+          max: number
+          avg: number
+        }
+        successes: {
+          max: number
+          avg: number
+        }
+        statName: string
+      }
+    | {
+        // total
+        attempts: number
+        // total
+        successes: number
+        statName: string
+      })[]
+}[]
+```
+
+# `/event/{eventKey}/matches`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  redAlliance: string[]
+  blueAlliance: string[]
+  // UTC date - match predicted time
+  time: string
+  // example: qm3
+  key: string
+}[]
+```
+
+# `/event/{eventKey}/star`
+
+## `PUT`
+
+### Request
+
+```ts
+type Data = boolean
+```
+
+
+### Response
+
+```ts
+type Data = null
+```
+
+# `/event/{eventKey}/stats`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  team: string
+  teleop: (
+    | {
+        attempts: {
+          max: number
+          avg: number
+        }
+        successes: {
+          max: number
+          avg: number
+        }
+        statName: string
+      }
+    | {
+        // total
+        attempts: number
+        // total
+        successes: number
+        statName: string
+      })[]
+  auto: (
+    | {
+        attempts: {
+          max: number
+          avg: number
+        }
+        successes: {
+          max: number
+          avg: number
+        }
+        statName: string
+      }
+    | {
+        // total
+        attempts: number
+        // total
+        successes: number
+        statName: string
+      })[]
+}[]
+```
+
+# `/event/{eventKey}/team/{team}/info`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  // only if they have future matches
+  nextMatch?: {
+    redAlliance: string[]
+    blueAlliance: string[]
+    // UTC date - match predicted time
+    time: string
+    // example: qm3
+    key: string
+  }
+  rank: number
+  // ranking score
+  rankingScore: number
+}
+```
+
+# `/event/{eventKey}/team/{team}/stats/auto`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  modeName: string
+  stats: (
+    | {
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+        attempts: number
+        successes: number
+        statName: string
+      }
+    | {
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+        attempted: boolean
+        succeeded: boolean
+        statName: string
+      })[]
+}[]
+```
+
+# `/event/{eventKey}/team/{team}/stats/teleop`
```

Comment:
It seems redundant to send which event the data is coming from since the endpoint already specifies that. As with the auto version of this, having all the data in one giant array seems like a not-great way to represent it.

Reply:
:heavy_check_mark:  on the removing event part, since it is duplicated

I think a giant array is fine